### PR TITLE
Basic layout with modal form and ability to save to database

### PR DESCRIPTION
**Pull request by mandytrinh**

### CODE REVIEW

File: db.js [hunk 1]
```diff
@@ -27,18 +27,37 @@ const getAllReviews = (productID, callback) => {
 }
 
 const saveAReview = (reviewObj, callback) => {
+  let photo_path = reviewObj.photo_path.slice(55);
```
Comment (jpcastberg) [inline db.js]: Curious to know what the purpose of slicing the photo path is

File: db.js [hunk 2]
```diff
@@ -27,18 +27,37 @@ const getAllReviews = (productID, callback) => {
...
+    reviewObj.product_ID
+  ]
+
   let queryStr = 'INSERT INTO reviews (user_ID, product_ID, rating, recommend, title, message, date_posted, photo_path, age_range) VALUES (?, ?, ?, ?, ?, ?, CURDATE(), ?, ?)';
+
+  let queryStr2 = 'INSERT INTO users (username, loc, age_range) VALUES (?, ?, ?)';
+
+  let queryStr3 = 'INSERT INTO photos (path, user_ID, product_ID) VALUES (?, ?,?)';
+
   connection.query(queryStr, values, (err) => {
```
Comment (jpcastberg) [inline db.js]: Nice, didn't know you could pass an array of values into a MySql query, but it makes sense!

**Comment by jpcastberg** [comment]
Nice job Mandy. I really like how clean your handleChange function is. Your code is also clear and readable.

File: client/components/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import $ from 'jquery';
+
+class Form extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user_ID: this.props.user_ID,
```
Comment (xtrabit) [inline client/components/Form.jsx]: As far as I understood, it is not a good practice to assign props to state directly.

File: client/components/reviews.jsx [hunk 4]
```diff
@@ -1,31 +1,122 @@
...
+    this.prevThreeIdx = this.currentIdx;
+    this.currentIdx = this.currentIdx + 3;
+    
+    this.setState({
+      currentReviews: this.state.allReviews.slice(this.prevThreeIdx, this.currentIdx),
+    });
+  }
+
+  handlePrev() {
+    if (this.currentReviews !== [] && this.currentIdx > 0 && this.prevThreeIdx > 0) {
```
Comment (xtrabit) [inline client/components/reviews.jsx]: If you are checking length of array here, is not [ ] === [ ] equals to false?

File: db.js (hunk 1 above)
Comment (xtrabit) [reply db.js]: I think to get item ID

**Pull request merged**